Library/Formula/hashcat.rb

```diff
@@ -0,0 +1,39 @@
+class Hashcat < Formula
+  desc "Advanced CPU-based password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://codeload.github.com/hashcat/hashcat/tar.gz/2.00"
+  sha256 "6325e6d75a4df3485adec00f74e5887326809c15ed31bfe74a12b62943245444"
+
+  head do
+    url "https://github.com/hashcat/hashcat.git"
+  end
+
+  depends_on "lzip"
+  depends_on "gcc" # with multilib
+  depends_on "gmp"
+
+  fails_with :clang do
+    cause <<-EOS.undent
+      This formula does not compile with clang, use GCC.
+    EOS
+  end
+  fails_with :llvm do
+    cause <<-EOS.undent
+      Upstream depends on GCC with multilib.
+    EOS
+  end
+
+  def install
+    # this flag disappeared on modern GCCs (was used only for apple-gcc42)
+    inreplace "src/Makefile", "-fnested-functions", ""
```

Comment:
Without this, apple-gcc42 is needed. Since it builds fine with a regular GCC on Linux, I suppose it's the same with our regular GCC from Homebrew.